Feature/update OCI docker documentation (#3)
idrsolutions/idrsolutions.github.io: _includes/cloud-docker/oraclecloud.md

One of the easiest ways to deploy {{ page.product }} on Oracle Cloud Infrastructure is to run a single compute instance and installing docker on it.

## Download the WAR file
Before getting started, you must first download the WAR file. This is **required** for the image to work.

**For trial users:**
1. If you haven't already, you can sign up for a [{{ page.product }} trial][trial-download].

2. Open the email you received when you signed up.

3. Follow the download link found in the docker section.
<p>&nbsp;</p>

**For customers:**
1. Visit the [customer download page][customer-download] for {{ page.product }}.

2. Click on 'Download Latest Customer WAR'.

> 3. Enter your username and password.
> 
> 4. Click submit.

Reviewer: ```suggestion
3. Submit your username and password.
```

Now launch the docker image.

```

> sudo docker run -p 80:80 --mount "source=/home/opc/{{ page.product | downcase }}_microservice.war,target=/usr/local/tomcat/webapps/ROOT.war,type=bind" idrsolutions/{{ page.product | downcase }}

Reviewer 2: Just checking here, not too sure but does it have to be tomcat in the target path or can it be like jetty, glassfish, etc. If so would it be worth noting this to the customer, like mount to the web app you set up.

Author: A good question! As this guide is specifically for using our provided docker images ([JPedal ](https://github.com/idrsolutions/jpedal-docker/blob/master/Dockerfile)/ [BuildVu](https://github.com/idrsolutions/buildvu-docker)) we can expect a set environment. If you check the Dockerfile in either project you'll see that the image is based on a Tomcat docker image, as such we only expect tomcat for the target path (at present the other app servers are not installed in the docker image).

Reviewer 2: ahh okay, thank you for explaining!

`private.key`
`ca_bundle.crt`

Now run the following command to start the service:

```
docker run -p 80:80 -p 433:433 --mount "source=/home/opc/{{ page.product | downcase }}_microservice.war,target=/usr/local/tomcat/webapps/ROOT.war,type=bind" idrsolutions\{{ page.product | downcase }}
```

If you encounter an error then you may need to run the following command instead. Please take caution in doing this - more information can be found in the [docker docs](https://docs.docker.com/storage/#configure-the-selinux-label).
```
docker run -p 80:80 -p 443:443 --mount "source=/home/opc/{{ page.product | downcase }}_microservice.war,target=/usr/local/tomcat/webapps/ROOT.war,type=bind" --volume "/path/to/ssl"/target:"/opt/ssl"/app:z,ro idrsolutions/{{ page.product | downcase }}
```

## Environment variables
There are a few useful environment variables that you can use.

> These are used to set a username and password for your users to login with:

Reviewer: ```suggestion
These are used to set a username and password to access the microservice:
```
